content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: '.getDisplayName()' 
+Description: 'Returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+ 
+Tags: 
+  - 'Date'
+  - 'Methods'
+  - 'Calender'
+CatalogContent: 
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+
+The **.getDisplayName()** method of the `Calender` class returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.
+
+## Syntax
+
+```pseudo
+public String getDisplayName(int field, int style, Locale locale)
+```
+
+```pseudo
+// Create a Calendar object.
+Calendar calendar = Calendar.getInstance();
+
+// Get the display name for the month in the current locale.
+String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+// Print the month name.
+System.out.println(monthName);
+```
+
+## Parameters
+
```

Comment:
```suggestion
```

Reply:
Again, I am not sure of the change here?

Reply:
Removed the parameter heading. It is expected that the syntax subsection will have parameter explanation.